README.md

```diff
@@ -608,6 +608,40 @@ br + br {
 
 <sup>[back to table of contents](#table-of-contents)</sup>
 
+### Multiple Borders 
+You can use box-shadow to make a border effect, by making the the shadow offset and have 0 blur. Plus, by comma-separating values, you can have as many "borders" as you like:
+
+```css
+div{
+   background-color:yellowgreen;
+   box-shadow:0 0 0 10px #656;
+}
+```
+This is not particularly impressive, as you can create the same kind of border
+by using the border property. However, the good thing about boxshadow is that we can have as many of them as we want, comma
+separated. So, we can pretty easily add a second deeppink “border”
+to the previous example:
+
+```css
+div{
+   background-color:yellowgreen;
+   box-shadow: 0 0 0 10px #655, 0 0 0 15px deeppink;
```

Comment:
Maybe easier to understand if you stay with color names for consistency? And unified spacing
```suggestion
   background-color: yellowgreen;
   box-shadow: 0 0 0 10px blue, 
               0 0 0 15px pink;
```